Core/OfficeDevPnP.Core/Framework/Graph/SiteClassificationUtility.cs

```diff
@@ -0,0 +1,269 @@
+﻿using Newtonsoft.Json;
+using OfficeDevPnP.Core.Framework.Graph;
+using OfficeDevPnP.Core.Framework.Graph.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OfficeDevPnP.Core.Framework.Graph
+{
+    public static class SiteClassificationUtility
```

Comment:
I would rename this one to SiteClassificationsUtility, if we made plural names in the extension methods, I would keep this one plural, too.